07_Train_With_AML_Pipeline.ipynb

```diff
@@ -47,10 +49,13 @@
     "from azureml.train.hyperdrive import (\n",
     "    RandomParameterSampling, choice, PrimaryMetricGoal,\n",
     "    HyperDriveConfig, MedianStoppingPolicy)\n",
-    "\n",
+    "from azureml.pipeline.core.schedule import ScheduleRecurrence, Schedule\n",
     "import azureml.core\n",
     "from get_auth import get_auth\n",
-    "print('azureml.core.VERSION={}'.format(azureml.core.VERSION))"
+    "print('azureml.core.VERSION={}'.format(azureml.core.VERSION))\n",
+    "\n",
+    "from MetricsUtils.hpStatisticsCollection import statisticsCollector, CollectionEntry\n",
+    "from MetricsUtils.storageutils import storageConnection"
```

Comment:
The published happy path deliberately does not create a scheduled pipeline. Doing so can leads to user problems and unexpected user costs. In addition, instrumentation should not be in the published path. These would be fine for the instrument branch.